extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -9,7 +9,9 @@ class ForumDumper {
 	const TABLE_REVISION = 'revision';
 	const TABLE_TEXT = 'text';
 	const TABLE_VOTE = 'page_vote';
-	const TABLE_PAGE_WIKIA_PROPS  = 'page_wikia_props';
+	const TABLE_PAGE_WIKIA_PROPS = 'page_wikia_props';
+	//TODO verify that this is the table
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)